src/utils/delseksjonUtils.ts

```diff
@@ -60,3 +63,34 @@ export const oppdaterFlettefeltIDelseksjonerState = (
     }
     return delseksjonStateKopi;
 };
+
+export const oppdaterFritekstTabellFraDelseksjonState = (
+    delseksjon: SanityDelseksjon,
+    delseksjonerState: StateDelseksjon[],
+    delseksjonIndeks: number
```

Comment:
Her kan du sende inn rett delseksjonState med en gang i stedet for å sende inn både delseksjonstate og delsesjonIndeks. Den "lookupen" kan du gjøre der metoden kalles.